p2p/conn/secret_connection.go

```diff
@@ -98,9 +111,14 @@ func MakeSecretConnection(conn io.ReadWriteCloser, locPrivKey crypto.PrivKey) (*
 	if err != nil {
 		return nil, err
 	}
+	io.ReadFull(hkdfHiEphPub, hdkf_state[:])
+
+	hkdfSecret := hkdf.New(hash, dhSecret[:], hdkf_state[:], []byte("TENDERMINT_SECRET_CONNECTION_TRANSCRIPT_HASH"))
+
+	io.ReadFull(hkdfSecret, hdkf_state[:])
 
 	// generate the secret used for receiving, sending, challenge via hkdf-sha2 on dhSecret
```

Comment:
Comment needs updating too. Something along the lines of?
```suggestion
	// generate the secret used for receiving, sending, challenge via hkdf-sha2 on the transcript state
```